panc/src/test/pan/Functionality/function/json1.tpl

```diff
@@ -0,0 +1,18 @@
+#
+# Test json decoding
+#
+# @expect="/nlist[@name='profile']/boolean[@name='boolean']='true'"
+# @expect="/nlist[@name='profile']/long[@name='long']='1234'"
+# @expect="/nlist[@name='profile']/double[@name='double']='1234.56'"
+# @expect="/nlist[@name='profile']/string[@name='string']='string'"
+# @expect="/nlist[@name='profile']/list[@name='list']/*[3]='three'"
+# @expect="/nlist[@name='profile']/nlist[@name='hash']/long[@name='b']='2'"
+#
+object template json1;
+
+"/boolean" = json_decode("true");
```

Comment:
for sanity reasons, could you also add a check like 
```pan
"/boolean" = json_encode(json_decode("true"));
```
and the inverse `json_decode(json_encode())`